rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -186,7 +186,11 @@ struct Cdds
   {}
 };
 
-static Cdds gcdds;
+static Cdds& gcdds()
```

Comment:
Uncrustify is cranky about this.

```suggestion
static Cdds & gcdds()
```